<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
  background-color: rgb(149, 249, 255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deposit amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tax rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deposit term:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Periodicity of payments:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Partial withdrawals:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Replenishments:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="deposit_term">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: white;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_term">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>63</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: white;
}</string>
   </property>
   <property name="currentText">
    <string>days</string>
   </property>
   <item>
    <property name="text">
     <string>days</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>years</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>101</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="periodicity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>186</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: white;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Every day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Every week</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Semiannually</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Once a year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>At the end of the term</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="capitalization">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Capitalization of interest:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_date">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color: white;
}</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start of term:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>27</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>104</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}

</string>
   </property>
   <property name="text">
    <string>Ð¡ALCULATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}

</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="wit_table">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
}
</string>
   </property>
   <property name="rowCount">
    <number>100</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QDateEdit" name="wit_date">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color: white;
}</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wit_amount">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="wit_add">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="rep_table">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
}
</string>
   </property>
   <property name="rowCount">
    <number>100</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
  </widget>
  <widget class="QDoubleSpinBox" name="rep_amount">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDateEdit" name="rep_date">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color: white;
}</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>10</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="rep_add">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  border: 1px solid grey;
  border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>41</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>11</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>73</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tax amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Accrued interest:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="deposit_table">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>321</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
	background-color: white;
	  border: 1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="accrued_interest">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_amount">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>73</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>64</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>42</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>â‚½</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>115</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>by the end of the term:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>amount</zorder>
  <zorder>deposit_term</zorder>
  <zorder>type_term</zorder>
  <zorder>interest_rate</zorder>
  <zorder>tax_rate</zorder>
  <zorder>periodicity</zorder>
  <zorder>capitalization</zorder>
  <zorder>start_date</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>calculate</zorder>
  <zorder>clear</zorder>
  <zorder>wit_table</zorder>
  <zorder>wit_date</zorder>
  <zorder>wit_amount</zorder>
  <zorder>wit_add</zorder>
  <zorder>rep_amount</zorder>
  <zorder>rep_date</zorder>
  <zorder>rep_add</zorder>
  <zorder>tax_amount</zorder>
  <zorder>label_12</zorder>
  <zorder>label_14</zorder>
  <zorder>label_13</zorder>
  <zorder>label_15</zorder>
  <zorder>deposit_table</zorder>
  <zorder>accrued_interest</zorder>
  <zorder>end_amount</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>rep_table</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
